src/test/resources/data/stubbed/creditsafeData.csv

```diff
@@ -0,0 +1,2 @@
+sortcode,accountno,name,postcode,csresult,accountexists
+662031,26344696,Howell - McDermott,PX8 8ZE,exact,yes
```

Comment:
can this be changed to random variables?

Reply:
- Sort code and account number need to be valid otherwise the mod check fails.
- csresult and accountexists need to be correct for data returned by stub to ensure substring checks for assessBusinessBankDetails pass.

We can randomise name and postcode.